.github/workflows/main.yml

```
@@ -69,72 +69,72 @@ jobs:
-        docker-images: false
-        swap-storage: true
 
-    - name: Checkout repository
-      uses: actions/checkout@v3
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          # this might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB
+          tool-cache: false
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)